frontend/public/co-fetch.ts

```diff
@@ -3,36 +3,19 @@ import { HttpError, RetryError } from '@console/dynamic-plugin-sdk/src/utils/err
 import { authSvc } from './module/auth';
 import { getActiveCluster } from '@console/dynamic-plugin-sdk/src';
 import storeHandler from '@console/dynamic-plugin-sdk/src/app/storeHandler';
-
-// set required headers for console
-const getCSRFToken = () => {
-  const cookiePrefix = 'csrf-token=';
-  return (
-    document &&
-    document.cookie &&
-    document.cookie
-      .split(';')
-      .map((c) => _.trim(c))
-      .filter((c) => c.startsWith(cookiePrefix))
-      .map((c) => c.slice(cookiePrefix.length))
-      .pop()
-  );
-};
+import { getCSRFToken } from '@console/dynamic-plugin-sdk/src/utils/fetch/console-fetch-utils';
 
 export const applyConsoleHeaders = (url, options) => {
-  if (options.method !== 'GET') {
-    const token = getCSRFToken();
-    if (options.headers) {
-      options.headers['X-CSRFToken'] = token;
-    } else {
-      options.headers = { 'X-CSRFToken': token };
-    }
+  const token = getCSRFToken();
+  if (options.headers) {
+    options.headers['X-CSRFToken'] = token;
+  } else {
+    options.headers = { 'X-CSRFToken': token };
   }
 
   // If the URL being requested is absolute (and therefore, not a local request),
   // remove the authorization header to prevent credentials from leaking.
-  if (url.indexOf('://') >= 0 && options.headers) {
-    delete options.headers.Authorization;
+  if (options.method === 'GET' || url.indexOf('://') >= 0) {
```

Comment:
The comment above is not correct, since we are not dropping the authorization header anymore